udp_audio error

```
In file included from src/esphome.h:27,
                 from src/main.cpp:3:
src/esphome/components/udp_audio/udp_audio.hpp:8:10: fatal error: esphome/components/network/ip_address.h: No such file or directory
```

Getting this error when trying to compile/use the udp_audio module - using esp-idf and not arduino, is that why?

--- a/components/udp_audio/__init__.py
+++ b/components/udp_audio/__init__.py
@@ -2,11 +2,10 @@ import esphome.codegen as cg
 import esphome.config_validation as cv
 from esphome.components import microphone
 from esphome.components.network import IPAddress
-
 from esphome.const import CONF_ID, CONF_IP_ADDRESS, CONF_MICROPHONE, CONF_PORT
 
 AUTO_LOAD = ["socket"]
-DEPENDENCIES = ["microphone"]
+DEPENDENCIES = ["microphone", "network"]
 
 udp_audio_ns = cg.esphome_ns.namespace("udp_audio")
 UDPAudioComponent = udp_audio_ns.class_("UDPAudioComponent", cg.Component)

